### DISCUSSION TITLE
Suggestion for spray cans?

### DISCUSSION THREAD

**Discussion by 1DontEx1st** [aderusha/DDD-Printable-Wall-Control-System#27] [Q&A]
Is there anything better than using U-hooks for hanging spray cans?

**Comment by aderusha** [comment]
How big are your cans?

Sorry, phrasing.  What’s the diameter of the cans you want to hold?

**Reply by 1DontEx1st** [reply]
I missed this update last year but ended up building a wooden spray paint rack using the plans from https://www.thehandymansdaughter.com/diy-spray-paint-storage-rack/
![image](https://user-images.githubusercontent.com/6410132/215356106-67181176-723c-4b3b-ab70-00706c91d78d.png)

**Comment by root-hal9000** [comment]
I am about to make a design for a compressed air can that would fit the bill. It's a bit different: imagine a piece that hugs the can, but only about 1/3 of the can's diameter and with a floor for the bottom of the can - but that's just to keep it straight, the actual holding force is embedded magnets. I made some generic spacer center with internal holes for embedded magnets that have been great for holding small tools and rulers, just applying the same concept. I have used magnets just straight on the pegboard to hold things, but they slide around and I am trying to get it neat. Anyway, I will make a PR and post a comment here when I do

**Comment by leviable** [comment]
[I bought this from wall control ](https://www.wallcontrol.com/slotted-metal-pegboard-aerosol-spray-can-holder-red/) when I bought my boards, before I realized I could print better hooks at home. It works well, and I suppose it could be 3d printed on its side. I could get its exact dimensions and angles If someone who knows CAD is interested in designing an stl.

![image](https://user-images.githubusercontent.com/11619428/215266226-f387a911-79b5-4c32-8d72-154098ae229e.png)

**Reply by root-hal9000** [reply]
I just saw this by random chance, but I just happen do have come up with my own solution, which involves embedded magnets. I made it for a compressed air can, but then it turns out that all the other spray cans, wd-40 cans, etc that I had around happen to have the same diameter. I will make a PR with it and post it on printables soon.
![IMG_20230212_203834](https://user-images.githubusercontent.com/2352828/218962343-7fb02acd-2e45-463a-b367-4588dc9f857e.jpg)

https://user-images.githubusercontent.com/2352828/218962354-04dba8d4-a720-4273-9300-e81534fb6c95.mp4
